src/coreclr/hosts/unixcoreruncommon/coreruncommon.cpp

```diff
@@ -16,7 +16,11 @@
 #include <string>
 #include <string.h>
 #include <sys/stat.h>
-#ifdef HAVE_SYS_SYSCTL_H
```

Comment:
A nit - I would slightly prefer keeping the `#ifdef HAVE_SYS_SYSCTL_H` intact and just add the other two headers under the __FreeBSD__ ifdef to have the include just once in there.